canonical/charmhub.io
# upgrade mermaid 11.0.0 -> 11.4.1

## Done
- Upgrade mermaid version from 11.0.0 to 11.4.1
## How to QA
- Go to https://charmhub-io-2044.demos.haus/postgresql-k8s/docs/e-flowchart-charm, https://charmhub-io-2044.demos.haus/mysql-k8s/docs/e-flowcharts and https://charmhub-io-2044.demos.haus/cos-lite/docs/explanation/logging?channel=latest/edge and check that the mermaid diagrams are displaying correctly with the texts
## Testing

## Issue / Card https://warthogs.atlassian.net/browse/WD-17915
Fixes # 

## Screenshots
